.eslintrc.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   root: true,
   // This tells ESLint to load the config from the package `eslint-config-custom`
-  extends: ['custom'],
+  extends: ['@penumbra-zone/eslint-config-custom'],
```

Comment:
Note: packages published externally need to have their dependencies published as well

Reply:
there's a couple options here which don't require publishing an eslint config package

1. transition to eslint flat config, so it becomes a workspace dependency
2. programmatically remove the eslint devDeps with a `prepack` configuration